pvl_hsu_soiling.m

```diff
@@ -0,0 +1,351 @@
+%      determine cleaning.
+%   [SR]=pvl_hsu_soiling(Time, Rain, RainThresh, Tilt, PM2_5, PM10, ModelType)
+%      Uses the time, rainfall amounts, tilt angle, particulate matter
+%      concentrations, and rain cleaning threshold to predict soiling rate
+%      using the model type selected by ModelType. Uses hourly rain
+%      accumulation to determine cleaning.
+%   [SR]=pvl_hsu_soiling(Time, Rain, RainThresh, Tilt, PM2_5, PM10, ModelType, RainAccPeriod)
+%      Uses the time, rainfall amounts, tilt angle, particulate matter
+%      concentrations, and rain cleaning threshold to predict soiling rate
+%      using the model type selected by ModelType. The user specifies the
+%      rain accumulation period (in hours) over which to determine whether
+%      sufficient rain has fallen to clean the module.
+%   [SR]=pvl_hsu_soiling(Time, Rain, RainThresh, Tilt, PM2_5, PM10, ModelType, RainAccPeriod, LUC, WindSpeed, Temperature)
+%      Uses the parameters of the fixed models along with Land Use Category
+%      (LUC), wind speed, and ambient temperature to predict soiling ratio
+%      using the variable deposition velocity model.
+%
+% Input Parameters:
+%   Time is a struct with elements as described in pvl_maketimestruct. 
+%     Time values for the soiling function do not need to be 
+%     regularly spaced, although large gaps in timing are discouraged.
+%
+%   Rain is a vector of rainfall values of the same length as the fields of
+%     Time, where each element of Rain correlates with the element of Time.
+%     Rainfall values should be in mm of rainfall. Programmatically, rain
+%     is accumulated over a given time period, and cleaning is applied
+%     immediately after a time period where the cleaning threshold is
+%     reached.
+%
+%   RainThresh is a scalar for the amount of rain, in mm, in a an accumulation
+%     period needed to clean the PV modules. In periods where the 
+%     accumulated rain meets or exceeds RainThresh, the panels are assumed 
+%     to be cleaned immediately after the accumulation period 
+%     [1] suggests a good RainThresh could be 1mm, but the time period is
+%     not specified. Rain accumulation period length can be adjusted in the
+%     optional input RainAccPeriod.
+%
+%   Tilt is a scalar or vector for the tilt of the PV panels. Changing tilt
+%     angles (e.g. in tracking cases) can be accomodated, and tilt angles
+%     are correlated with the entries in Time.
+%
+%   PM2_5 is the concentration of particulate matter with diamter less than
+%     2.5 microns, in g/m^3. Historical US concentration data sets may be
+%     found at https://aws.epa.gov/aqsweb/airdata/download_files.html. With
+%     file descriptions (for hourly data sets) at 
+%     https://aqs.epa.gov/aqsweb/airdata/FileFormats.html#_hourly_data_files
+%   
+%   PM10 is the concentration of particulate matter with diamter less than
+%     10 microns, in g/m^3. Historical US PM data sets found as described
```

Comment:
what is "PM"

Reply:
Corrected to indicate that PM is the abbreviation for Particulate Matter.